tests/wasm/getters_and_setters.rs

```diff
@@ -32,6 +34,22 @@ pub struct Rules {
     pub field: i32,
 }
 
+#[wasm_bindgen]
+pub struct RulesWithRawField {
+    pub r#field: i32,
```

Comment:
While we are at it, would you mind using a keyword just to test an actual use-case?
E.g.:
```suggestion
    pub r#mod: i32,
```

Reply:
@daxpedda sure! I am not sure if the test I wrote is correct or not, it seems we only need to test if `r#mod` field expanded by `#[wasm_bindgen]` macro is correct or not, so the beblow section seems unnecessary.
```
impl RulesWithRawField {
    #[wasm_bindgen]
    pub fn get_field_value(&self) -> i32 {
        self.r#field
    }
    #[wasm_bindgen]
    pub fn set_field_value(&self, value: i32) {
        self.r#field = value;
    }
}
// ...

```

Reply:
I think we can leave this here, just to double check if the generated code in the methods can also handle raw identifiers.